qiita_db/data.py

```diff
@@ -498,11 +512,11 @@ def remove_filepath(self, fp):
             raise e
 
         # We can already update the status to done, as the files have been
-        # unlinked, the purge_filepaths call will not change the status
-        # of the raw data object
+        # unlinked, the move_filepaths_to_upload_folder call will not change
+        # the status of the raw data object
         self._set_link_filepaths_status("idle")
 
-        # Delete the file if it is not used anywhere
+        # Delete the files, if they are not used anywhere
         purge_filepaths(conn_handler)
```

Comment:
Sorry for not commenting this earlier, but this function and the one above are very similar, the only difference is that `remove_filepath` only removes one filepath, while `clear_filepaths` removes all of them. In that case, should this function use the `move_filepaths_to_upload_folder` instead of the `purge_filepaths` one?